app/javascript/components/organisms/PageHeader.js

```diff
@@ -13,7 +13,7 @@ const PageHeader = ({ title, content, action, item, itemA, itemB }) => {
               <p>{content}</p>
             </div>
             <div className="medium-4 columns no-padding">
-              <span className="lead-header_secondary-action"><a href="#" class="alt-caps">{action}</a></span>
+              <span className="lead-header_secondary-action"><a href="#" className="alt-caps">{action}</a></span>
```

Comment:
@slowbot just a heads up for future PL components markup that React uses className instead of class